src/commands.c

```diff
@@ -1412,12 +1548,116 @@ int yh_com_open_session(yubihsm_context *ctx, Argument *argv, cmd_format fmt) {
   return 0;
 }
 
+// NOTE(adma): Open a session with a connector using an Asymmetric
+// Authentication Key argc = 2 arg 0: w:authkey arg 1: i:password
```

Comment:
Please follow the format for other commands laying out what the arguments look like

Reply:
I did, but pre-commit reverts it to this format

Reply:
Removed the adma, See commit 23c464a8f8c92445ad4b85c271abafac5eb6e0ce